helpers/response_parser.py

```diff
@@ -0,0 +1,69 @@
+"""
+Module to parse curl-like response.
```

Comment:
I believe it would be better to have this doc-string directly within the `Response` class, this way IDE's have a better chance to provide more related autogenerated docs. Also, the name `Response` doesn't point out the actual purpose of this class and leads to confusion and collisions with, for example, `helpers.deproxy.Response`, so it perhaps should be renamed to something like `helpers.curl.CurlResponse`.

Reply:
renamed to CurlResponse